controller/src/main/java/io/pravega/controller/store/client/ZKClientConfig.java

```diff
@@ -47,4 +47,18 @@
      */
     int getSessionTimeoutMs();
 
+    /**
+     * Fetches whether the connection is secure.
+     */
+    boolean isSecureZK();
```

Comment:
`isSecureZK`  is odd. Can we be more descriptive, like `isZooKeeperCnxnSecure`?

Reply:
Fixed. Using `isConnectionToZooKeeperSecure`.